cloud/pkg/devicecontroller/controller/downstream.go

```diff
@@ -410,6 +392,9 @@ func (dc *DownstreamController) deviceAdded(device *v1alpha2.Device) {
 		if err != nil {
 			klog.Errorf("Failed to send device addition message %v due to error %v", msg, err)
 		}
+
+		dc.sendDeviceModelMsg(device, model.InsertOperation)
+		dc.sendDeviceMsg(device, model.InsertOperation)
```

Comment:
i'm confused about the process of new message and old message. here if we send dmi msg to the edge, the old msg is also sent, will not it cause edgecore create one device twice for one device?

Reply:
edgecore will only create one device for once but it will send the message to mapper both through mqtt and dmi. there is only one mapper on the edgenode and it will only process one message with mqtt or dmi at the same time.